client/src/components/FieldHelper.js

```diff
@@ -152,12 +153,13 @@ export const InputField = ({
   const widgetElem = useMemo(
     () => (
       <FormControl
+        type={inputType || null}
```

Comment:
Is the ` || null` strictly necessary here?

Reply:
Hmm i guess not